config/dynamic.js

```diff
@@ -0,0 +1,104 @@
+const path = require("path");
+
+const apolloBatching = process.env.APOLLO_BATCH !== 'false';
+const memcachedServers = process.env.MEMCACHE_HOST || 'ui-memcached:11211';
+const baseUrl = process.env.BASE_URL || 'development.kiva.org';
+const env = process.env.SHORT_ENV || 'dev';
+const formattedUrlEnv = env === 'prod' ? '' : `${env}.`;
+const enableOptimizely = process.env.ENABLE_OPTIMIZELY !== 'false';
+const enablePerimeterx = process.env.ENABLE_PERIMETERX !== 'false';
+const disableCluster = process.env.SERVER_DISABLE_CLUSTER === 'true';
```

Comment:
This one can just be set to `true` now. We always disable cluster most in k8s infra.